core/src/main/resources/defaults/blueprints/cdp-opdb-custom.json

```diff
@@ -0,0 +1,158 @@
+{
+  "description": "CDP 1.0 Operational Database with Apache HBase (custom)",
+  "blueprint": {
+    "cdhVersion": "7.0.0",
+    "displayName": "opdb-custom",
+    "services": [
+      {
+        "refName": "zookeeper",
+        "serviceType": "ZOOKEEPER",
+        "roleConfigGroups": [
+          {
+            "refName": "zookeeper-SERVER-BASE",
+            "roleType": "SERVER",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "hdfs",
+        "serviceType": "HDFS",
+        "roleConfigGroups": [
+          {
+            "refName": "hdfs-NAMENODE-BASE",
+            "roleType": "NAMENODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-SECONDARYNAMENODE-BASE",
+            "roleType": "SECONDARYNAMENODE",
+            "configs": [
+              {
+                "name": "fs_checkpoint_dir_list",
+                "value": "/should_not_be_required_in_HA_setup"
+              }
+            ],
+            "base": true
+          },
+          {
+            "refName": "hdfs-DATANODE-BASE",
+            "roleType": "DATANODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-FAILOVERCONTROLLER-BASE",
+            "roleType": "FAILOVERCONTROLLER",
+            "base": true
+          },
+          {
+            "refName": "hdfs-JOURNALNODE-BASE",
+            "roleType": "JOURNALNODE",
+            "base": true
+          },
+          {
+            "refName": "hdfs-GATEWAY-BASE",
+            "roleType": "GATEWAY",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "hbase",
+        "serviceType": "HBASE",
+        "serviceConfigs": [
+          {
+            "name": "hbase_wal_dir",
+            "value": "/hbase-wals"
+          }
+        ],
+        "roleConfigGroups": [
+          {
+            "refName": "hbase-MASTER-BASE",
+            "roleType": "MASTER",
+            "base": true
+          },
+          {
+            "refName": "hbase-GATEWAY-BASE",
+            "roleType": "GATEWAY",
+            "base": true
+          },
+          {
+            "refName": "hbase-REGIONSERVER-BASE",
+            "roleType": "REGIONSERVER",
+            "configs": [
+              {
+                "name": "hbase_regionserver_maxlogs",
+                "value": "100"
+              }
+            ],
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "knox",
+        "serviceType": "KNOX",
+        "roleConfigGroups": [
+          {
+            "base": true,
+            "refName": "knox-KNOX-GATEWAY-BASE",
+            "roleType": "KNOX_GATEWAY",
+            "configs": [
+              {
+                "name": "gateway_dispatch_whitelist",
+                "value": "^https?:\\/\\/.+\\.cloudera\\.site:[0-9]+.*$"
+              },
+              {
+                "name": "gateway_knox_admin_groups",
+                "value": "admins"
```

Comment:
I don't think we need this property as it will change per env to be cdp_$env_name

Reply:
Yeah, I can drop this one now, thanks.